### DISCUSSION TITLE
Autofocus on password input in WebUI

### DISCUSSION THREAD

**Discussion by Nexulo** [dani-garcia/vaultwarden#3732] [Ideas]
When accessing Vaultwarden via WebUI, you have to enter your email address in the first login screen and the cursor is already automatically placed in the input field. 

Unfortunately, when you get to the password input field, the cursor is not automatically placed in the input field. Can this please be adjusted so that the cursor is also automatically set in the password field? This way you can login a little bit faster.

**Comment by BlackDex** [comment]
First, this is not something Vaultwarden maintains. The web-vault is not managed by this project.
Second, it looks like this is fixed in 2023.7.0, which is not yet part of our released versions.

**Comment by Nexulo** [comment]
Ah okay I apologize very much. Thanks for the info.
